actionpack/lib/action_dispatch/system_test_case.rb

```diff
@@ -105,9 +105,12 @@ def self.start_application # :nodoc:
     #
     #   driven_by :selenium, screen_size: [800, 800]
     def self.driven_by(driver, using: :chrome, screen_size: [1400, 1400])
-      SystemTesting::Driver.new(driver).run
+      driver = selenium?(driver) ? SystemTesting::Browser.new(using, screen_size) : SystemTesting::Driver.new(driver)
```

Comment:
Weird. One of my comments disappeared...

Can you change this into a multi-line method that's called from driven_by?